src/schemas/helloWorld.js

```diff
@@ -1,9 +1,9 @@
-const { gql } = require('apollo-server')
+const { gql } = require('apollo-server');
```

Comment:
I noticed that sometimes we're using `gql` from `apollo-server` and other times we're importing it from `apollo-server-express`, not sure if we want to change it for consistency so we only use one package instead

Reply:
I'm not sure what the difference is, but it's probably a good idea to import all from the same place (it seem like `apollo-server-express` would be the one) if it's all doing the same thing functionally

I skimmed through https://www.apollographql.com/docs/apollo-server/migration-two-dot/ but it's not all that clear to me lol